cmd/virtual-kubelet-backend-titus-executor/main.go

```diff
@@ -89,9 +90,16 @@ func mainWithError(podFileName string, statusPipe string, dockerCfg *docker.Conf
 	defer pipe.Close()
 	log.G(ctx).WithField("pod", pod.Name).Debugf("Got pipe %v", statusPipe)
 
-	logUploaders := uploader.NewUploadersFromUploaderArray([]uploader.Uploader{})
+	m := metrics.Discard
 
-	dockerRunner, err := runner.New(ctx, metrics.Discard, logUploaders, *cfg, *dockerCfg)
+	log.G(ctx).WithField("pod", pod.Name).Debugf("Getting uploaders from %+v", cfg.S3Uploaders)
+	var logUploaders *uploader.Uploaders
+	if logUploaders, err = uploader.NewUploaders(cfg, m); err != nil {
+		return fmt.Errorf("cannot create log uploaders: %v", err)
```

Comment:
use errors.wrapf

Reply:
err, errors.wrap

Reply:
[You first](https://github.com/Netflix/titus-executor/blob/master/cmd/titus-executor/main.go#L119-L142).